### DISCUSSION TITLE
Some plots are not recognized other are in the same root but mostly on the same HD

### DISCUSSION THREAD

**Discussion by Motopoeta1** [Chia-Network/chia-blockchain#8177] [Support]
I have a strange issue. I have two list of plots in two hard disks fully dedicated to CHIA. In another one, with other file inside, I have 20 plots in the same folder and only 1 of them is recognized, all the other plot are not. Why?

**Comment by diaconom** [comment]
One of the best ways to check is to run: chia plots check -n 0 - l

This will check for duplicates first. If no duplicate run same command without any parameters ie: chia plots check

This will take more time as it runs about 30 tests on each plot but at the end you will see if you have any errors.

Good luck.

**Reply by Motopoeta1** [reply]
great, solved! thanks

**Comment by BrandtH22** [comment]
@Motopoeta1 , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
